it/src/test/java/org/corfudb/universe/GenericIntegrationTest.java

```diff
@@ -1,111 +1,70 @@
 package org.corfudb.universe;
 
-import com.google.common.collect.ImmutableSet;
-import com.spotify.docker.client.DefaultDockerClient;
-import com.spotify.docker.client.DockerClient;
-import org.corfudb.universe.logging.LoggingParams;
-import org.corfudb.universe.scenario.Scenario;
-import org.corfudb.universe.scenario.fixture.Fixtures.AbstractUniverseFixture;
-import org.corfudb.universe.scenario.fixture.Fixtures.UniverseFixture;
-import org.corfudb.universe.scenario.fixture.Fixtures.VmUniverseFixture;
-import org.corfudb.universe.universe.Universe;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.corfudb.universe.UniverseManager.UniverseWorkflow;
+import org.corfudb.universe.node.NodeException;
+import org.corfudb.universe.scenario.fixture.Fixture;
 import org.corfudb.universe.universe.Universe.UniverseMode;
 import org.corfudb.universe.universe.UniverseParams;
-import org.corfudb.universe.universe.vm.ApplianceManager;
-import org.corfudb.universe.universe.vm.VmUniverseParams;
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.rules.TestName;
 
-import java.util.Set;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.function.Consumer;
 
 public abstract class GenericIntegrationTest {
```

Comment:
Comments.  What is this suppose to test?

Reply:
done